pygeoapi/process/manager/postgresdb_.py

```diff
@@ -0,0 +1,261 @@
+                cur.execute(query_update, update_dict)
+                rowcount = cur.rowcount
+                conn.commit()
+
+        return rowcount == 1
+
+    def get_job(self, job_id: str) -> dict:
+        """
+        Get a single job
+
+        :param job_id: job identifier
+
+        :raises JobNotFoundError: if the job_id does not correspond to a
+                                  known job
+        :returns: `dict`  # `pygeoapi.process.manager.Job`
+        """
+
+        with self.get_db_connection() as conn:
+            with conn.cursor(cursor_factory=psycopg2.extras.DictCursor) as cur:
+                query_select = \
+                    """SELECT * FROM job_info WHERE identifier = %s"""
+                query_params = [job_id]
+                cur.execute(query_select, query_params)
+                found = cur.fetchone()
+
+        if found is not None:
+            return found
+        else:
+            raise JobNotFoundError()
+
+    def delete_job(self, job_id: str) -> bool:
+        """
+        Deletes a job
+
+        :param job_id: job identifier
+
+        :raises JobNotFoundError: if the job_id does not correspond to a
+                                  known job
+        :return `bool` of status result
+        """
+        # delete result file if present
+        job_result = self.get_job(job_id)
+        location = job_result.get('location')
+        if location and self.output_dir is not None:
+            try:
+                Path(location).unlink()
+            except FileNotFoundError:
+                pass
+
+        query_delete = "DELETE FROM job_info WHERE identifier = %s"
+        with self.get_db_connection() as conn:
+            with conn.cursor() as cur:
+                cur.execute(query_delete, [job_id])
+                rowcount = cur.rowcount
+                conn.commit()
+
+        return rowcount == 1
+
+    def get_job_result(self, job_id: str) -> Tuple[str, Any]:
+        """
+        Get a job's status, and actual output of executing the process
+
+        :param job_id: job identifier
+
+        :raises JobNotFoundError: if the job_id does not correspond to a
+                                  known job
+        :raises JobResultNotFoundError: if the job-related result cannot
+                                        be returned
+        :returns: `tuple` of mimetype and raw output
+        """
+
+        job_result = self.get_job(job_id)
+        location = job_result.get('location')
+        mimetype = job_result.get('mimetype')
+        job_status = JobStatus[job_result['status']]
+
+        if not job_status == JobStatus.successful:
+            # Job is incomplete
+            return (None,)
+        if not location:
+            LOGGER.warning(f'job {job_id!r} -  unknown result location')
+            raise JobResultNotFoundError()
+        else:
+            try:
+                location = Path(location)
+                with location.open('r', encoding='utf-8') as filehandler:
```

Comment:
We don't need `'r'` as it is the the default mode for file opening.

As well, suggest to replace `filehandler` with `fh` (common convention in codebase).